dev/io.openliberty.microprofile.openapi.2.0.internal_fat/fat/src/io/openliberty/microprofile/openapi20/fat/utils/OpenAPITestUtil.java

```diff
@@ -404,4 +406,96 @@ private static ObjectNode getFieldObject(JsonNode parent,
         assertTrue(childNode.isObject());
         return (ObjectNode) childNode;
     }
+
+    /**
+     * Checks if two JsonNode objects are recursively equal, ignoring the order of object properties
+     * 
+     * @param n1 the first node to compare
+     * @param n2 the second node to compare
+     * @return whether they are equal ignoring property order
+     */
+    public static boolean equalIgnoringPropertyOrder(JsonNode n1, JsonNode n2) {
+        if (n1 == n2) {
+            return true;
+        }
+
+        if (n1 == null || n2 == null) {
+            return false;
+        }
+
+        if (n1.getNodeType() != n2.getNodeType()) {
+            return false;
+        }
+
+        if (n1.isArray()) {
+            ArrayNode a1 = (ArrayNode) n1;
+            ArrayNode a2 = (ArrayNode) n2;
+            if (a1.size() != a2.size()) {
+                return false;
+            }
+            for (int i = 0; i < a1.size(); i++) {
+                if (!equalIgnoringPropertyOrder(a1.get(i), a2.get(i))) {
```

Comment:
It feels correct to me that "ignoring property order" does not mean "ignoring array contents order".

But worth checking that's the case?

Reply:
Yes, that's what I thought too. The order of array elements is still checked.